### ISSUE TITLE
Merge firefox-selection-fix

### ISSUE THREAD

**Issue by allo-** [black7375/Firefox-UI-Fix#32]
Since some time, Firefox has the annoying feature to select the text in the URL bar, when you click it. There is a fix here:
https://github.com/SebastianSimon/firefox-selection-fix

It looks like you could merge your projects into some project that allows to select which features to restore trying to maintain several fixes to the Firefox UI, that were created when the changes were introduced.

**Comment by black7375** [comment]
Awesome.
It's definitely worth a try and looks good.
It should be included in the next release.

**Comment by black7375** [comment]
It also helps when merging [branch/interactive-js](
https://github.com/black7375/Firefox-UI-Fix/tree/interactive-js) later.

Main use cases of this branch (Just concepts)
- Fluent Tab Hover Effect https://github.com/black7375/Firefox-UI-Fix/issues/2#issuecomment-843867954
- Restore Bookmark Animation
- Restore Page Action & Screenshot
- Battery Saver
- ...

**Comment by ojaha065** [comment]
> It should be included in the next release.

Personally I think that Firefox default behaviour selecting the whole URL when clicked is fine. I'm used to it working that way and wouldn't like it to change. But I totally understand why some people hate that feature. (Which is exactly why it was a foolish decision by Mozilla to remove the `browser.urlbar.clickSelectsAll` option.)

My point being that if something like this is implemented or merged to this project, it shouldn't be enabled by default and/or there should be an easy way to disable it.

**Comment by 7k5x** [comment]
@ojaha065 Me too. I'll implement prompting the user for selection.

**Comment by black7375** [comment]
@ojaha065 That's right. I think it's a possibility to activate the option and install it.
